package.json

```diff
@@ -23,8 +23,17 @@
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava"
   },
   "scriptsArgs": {
-    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{packages,scripts,.github}/**/*.{js,md,html}\"",
-    "prettier": "--ignore-path .gitignore --loglevel=warn \"{packages,scripts,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\" \"!**/package-lock.json\" \"!package-lock.json\""
+    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{packages,scripts,.github}/**/*.{js,md,html}\" \"*.{js,md,html}\" \".*.{js,md,html}\"",
+    "prettier": "--ignore-path .gitignore --loglevel=warn \"{packages,scripts,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\" \".*.{js,yml,json,html}\" \"!**/package-lock.json\" \"!package-lock.json\""
+  },
+  "ava": {
+    "files": [
+      "packages/**/tests/*.js",
+      "packages/**/tests/**/tests.js"
+    ],
+    "compileEnhancements": false,
+    "babel": false,
+    "verbose": true
```

Comment:
`ava.config.js` must be a ES module, which conflicts with linting everything as CommonJS. Putting it in `package.json` solves this.